src/index.js

```diff
@@ -4,6 +4,22 @@ import bindActionCreators from './utils/bindActionCreators'
 import applyMiddleware from './utils/applyMiddleware'
 import compose from './utils/compose'
 
+/*
+* create a function so that we can check if the function name has been altered by minification
+* if the function has been minified and NODE_ENV !== 'production', warn the user
+*/
+function isCrushed() {}
+
+if (isCrushed.name !== 'isCrushed' && process.env.NODE_ENV !== 'production') {
```

Comment:
Out of curiosity though I would expect to see conditionals with `process.env.NODE_ENV` or similar in the react codebase, but I don't. If I'm using webpack and requiring react how does defining it make the react build faster?